### ISSUE TITLE
Why I cannot edit the example code?

### ISSUE THREAD

**Issue by MrWanter** [fchollet/keras#2716]
Recently I have installed keras and run the example code of lstm for text generation succssefully, I want to change the model from LSTM to RNN, but when I change the import code, it cannot run, specifically,
in the example code, there is :
`from __future__ import print_function
from keras.models import Sequential
from keras.layers import Dense, Activation, Dropout
from keras.layers import LSTM
from keras.utils.data_utils import get_file
import numpy as np
import random
import sys`

but after I change `from keras.layers import LSTM` to `from keras.layers import SimpleRNN`, the spyder IDE indicates redline under 'SimpleRNN' and cannot work,I'm really confused, can anyone help me, thanks in advance!
